src/backends/http.ts

```diff
@@ -9,6 +9,9 @@ export default async function fetchHTTP(config: HTTPConfig): Promise<string> {
     const response = await axios.get(config.http_url, {
       method: 'get',
       responseType: 'stream',
+      headers: {
```

Comment:
Would you mind checking of config.authorization exists and only then passing in headers? Could be something like this:

```
const auth = { 
  headers: {
    authorization: config.authorization,
  }
};

const headers = config ? auth : {}

const response = await axios.get(config.http_url, {
  method: 'get',
  responseType: 'stream',
  ...headers,
})
```

Reply:
Yeah, I was wondering if I should do that. Good point. I'll add that in and push a new commit.